packaging/make-git-snapshot.sh

```diff
@@ -29,4 +29,15 @@ ls -al ${TARFILE_TMP}
     tar -A -f ${TARFILE_TMP} submodule.tar
     rm submodule.tar
 done
+tmpd=$(mktemp -d)
```

Comment:
Let's add a trap to nuke this for good measure? (I just tried this patch locally, and it died on some local config issue and left the directory behind!)